.travis.yml

```diff
@@ -33,16 +37,8 @@ matrix:
           - tox --installpkg ./dist/djangorestframework-*.whl
           - tox  # test sdist
 
-    exclude:
-      - { python: "2.7", env: DJANGO=master }
-      - { python: "2.7", env: DJANGO=2.0 }
-      - { python: "2.7", env: DJANGO=2.1 }
-      - { python: "3.4", env: DJANGO=master }
-      - { python: "3.4", env: DJANGO=2.1 }
-
```

Comment:
I'm finding the change in `.travis.yml` hard to follow with these other bits moved around. Is it possible to add 3.7 to the matrix in a more minimal way?

Reply:
I do want to reorganize the matrix though, as it is difficult to follow the combination of include, exclude, and matrix expansion. The single include section is a little more straightforward to follow.

Reply:
Why doesn't just adding to the `python` field work...?

```
python:
  - "3.4"
  - "3.5"
  - "3.6"
  - "3.7-dev"
```

A la [what we did on django-filter](https://github.com/carltongibson/django-filter/blob/8ca7e0c78550813d6d60a683840e9361d5c4610f/.travis.yml#L4-L8) 


>  If preferred, I can split the matrix reorg into another PR.

That might be worth it — mostly because of the fix in `test_model_serializer.py`

Reply:
3.7-dev is very outdated. In order to use the official release, you need to install with Ubuntu xenial+`sudo: true`. The workaround is covered in https://github.com/travis-ci/travis-ci/issues/9815